COMETwebapp/Components/Shared/ParameterTypeEditors/BooleanParameterTypeEditor.razor

```diff
@@ -30,6 +30,11 @@ along with this program. If not, see http://www.gnu.org/licenses/.
             Value="@this.ViewModel.ValueArray[this.ViewModel.ValueArrayIndex]"
             ValueChanged="@((string result) => this.ViewModel.OnParameterValueChanged(result))"
             ReadOnly="this.ViewModel.IsReadOnly" />
+
+    if (!string.IsNullOrEmpty(this.ViewModel.ValidationMessage))

```

Comment:
You could keep a special component to display the error message

Reply:
What I meant here is that your component was relevant, but use only a string as parameter instead of a VM has previously